README.md

```diff
@@ -1 +1,134 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
```

Comment:
- [ ] In your readme file kindly add the **Future Features** section and outline expected features for the app.

Reply:
@julie-ify thank you for your valuable suggestion. I am going to implement the changes.